CriticalMass/AppController.swift

```diff
@@ -38,6 +39,14 @@ class AppController {
         return NetworkOperator(networkIndicatorHelper: NetworkActivityIndicatorHelper(), dataProvider: session)
     }()
 
+    private let networkObserver: NetworkObserver? = {
```

Comment:
it seems like the networkObserver isn't injected into the RequestManager

Reply:
My fault, fixed